app/screens/select_server/select_server.js

```diff
@@ -227,11 +236,15 @@ class SelectServer extends PureComponent {
             this.cancelPing = null;
         };
 
-        this.props.actions.getPing().then((result) => {
+        getPing().then((result) => {
             if (cancel) {
                 return;
             }
 
+            if (!result.error && !Object.keys(config).length) {
+                loadConfigAndLicense();
```

Comment:
I think we might end up loading these twice when logging into a fresh server because it's also loaded by [this](https://github.com/mattermost/mattermost-mobile/blob/master/app/mattermost.js#L232). Maybe we disable that one when we're not logged in and then always call it here so that we're explicit about it?

Reply:
sounds good to me, will do that